### DISCUSSION TITLE
Is it avaliable on windows

### DISCUSSION THREAD

**Discussion by kylin419** [Spotifyd/spotifyd#1206] [Q&A] [answered]
Can spotifyd be download and run on windows system?

**Comment by slondr** [comment, marked as answer]
> Can spotifyd be download and run on windows system?

Hypothetically, yeah, but I don't believe it does right now. 

#602 was one attempt at making it work, but that effort seems to have been more or less abandoned.

**Reply by kylin419** [reply]
Are there any tools that can run on windows system to get the similar effect like spotifyd?

**Reply by slondr** [reply]
Not that I know of

**Reply by eladyn** [reply]
I _think_, you can run bare [`librespot`](https://github.com/librespot-org/librespot) on windows. (Although you'll have to compile it yourself.) This doesn't give you the features that `spotifyd` provides on top, but most of them are geared toward Unix systems anyway.

**Reply by kylin419** [reply]
> I _think_, you can run bare [`librespot`](https://github.com/librespot-org/librespot) on windows. (Although you'll have to compile it yourself.) This doesn't give you the features that `spotifyd` provides on top, but most of them are geared toward Unix systems anyway.

How can I use it?

**Reply by eladyn** [reply]
Make sure that you have [rust installed and setup](https://rustup.rs) and then run `cargo install librespot`. Then you should be able to run `librespot` from cmd.
